README.md

```diff
@@ -1,2 +1,214 @@
+
+You can also check linter errors by running this commands:
+
+```
+npx stylelint "**/*.{css,scss}"
+```
+
+```
+npx eslint "**/*.{js,jsx}"
+```
+
+### Usage <a name="usage"></a>
+
+Launch the app with the following command
+
+```
+  npm start
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Mohamed Aden Ighe**
+
+- GitHub: [@katarighe](https://github.com/katarighe)
+- Twitter: [@katarighe](https://twitter.com/katarighe)
+- LinkedIn: [@mighe](https://linkedin.com/in/mighe)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+Here are *some future features* that could be added to the Mini World Atlas Webapp in the future.
+
+- **Interactive maps:** The maps in the Mini World Atlas Webapp will be made more interactive, allowing users to zoom in and out, pan around, and explore different countries in more detail.
+
+- **More detailed country profiles:** The country profiles in the Mini World Atlas Webapp will be expanded to include more information about each country, such as its history, culture, economy, and government.
+
+- **New languages:** The Mini World Atlas Webapp will be translated into more languages, making it accessible to a wider audience.
+
+- **More data sources:** The Mini World Atlas Webapp will be powered by more data sources, providing users with the most up-to-date information about countries.
+
+- **Collaboration tools:** The Mini World Atlas Webapp will be made more collaborative, allowing users to share information and collaborate on projects.
+
+Moreover, here are some other possible features that could be added in the future:
+
+- A **search bar** to make it easier to find information about specific countries.
+
+- The ability to **add notes and comments** to country profiles.
+
+- The ability to **create custom maps**.
+
+- The ability to **export data** from the Mini World Atlas Webapp.
+
+The following are a few ideas for future features that could be added to the Mini World Atlas Webapp. The specific features that are added will depend on the feedback and suggestions of users.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+Give a star⭐️ or a thumbs up 👍 if you like this project! You can visit my GitHub profile for more of my projects.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- Project supervised by [Microverse](https//www.microverse.org/)
+- [Creative Commons 4.0 License](https://creativecommons.org/licenses/by-nc/4.0/)
```

Comment:
- [ ] Your credit should be ascribed to Nelson Sakwa directly who inspired the project design, not just Creative Commons License.

Reply:
✅ Done